### PR TITLE
Added Workshop Program

### PR DESCRIPTION
**Pull request by jgwinnup**
Added workshop program - one author missing but will be a quick fix

### CODE REVIEW

File: 2023/index.md [hunk 1]
```diff
@@ -99,10 +99,113 @@ The 3rd NLP-OSS workshop will be co-located with the EMNLP 2023 conference.
...
-<!-- ## Workshop Program
+## Workshop Program
 <br><br>
 
-The timezone for the program schedule below are in **Singapore Time (GMT +8)**.  -->
+The timezone for the program schedule below are in **Singapore Time (GMT +8)**.
+
+<p>09:00 - 09:15  &nbsp;&nbsp; <strong> Opening Remarks </strong> </p>
+<p>09:15 - 10:15 &nbsp;&nbsp; <strong> Invited Talk  - SEA-LION (Southeast Asian Languages In One Network): A Family of Southeast Asian Language Models </strong> <br>
+<em>William Tjhi, David Ong and Peerat Limkonchotiwat</em></p>
```
Comment (alvations) [inline 2023/index.md:109]: For this invited talk, we've confirmed only David and Peerat. William and other SEA-LION collaborators are busy so they will not be able to present. So I think for now, we'll have David and Peerat only.

But William and others from SEA-LION might attend the workshop, we should gladly introduce them if they do =)

File: 2023/index.md [hunk 2]
```diff
@@ -99,10 +99,113 @@ The 3rd NLP-OSS workshop will be co-located with the EMNLP 2023 conference.
...
 
-<!-- ## Workshop Program
+## Workshop Program
 <br><br>
 
-The timezone for the program schedule below are in **Singapore Time (GMT +8)**.  -->
+The timezone for the program schedule below are in **Singapore Time (GMT +8)**.
+
+<p>09:00 - 09:15  &nbsp;&nbsp; <strong> Opening Remarks </strong> </p>
+<p>09:15 - 10:15 &nbsp;&nbsp; <strong> Invited Talk  - SEA-LION (Southeast Asian Languages In One Network): A Family of Southeast Asian Language Models </strong> <br>
```
Comment (alvations) [inline 2023/index.md:108]: I think we have trlX first. 

```
<p>09:15 - 10:15 &nbsp;&nbsp; <strong>Invited Talk - trlX: A Framework for Large Scale Open Source RLHF </strong><br>
<em>Louis Castricato</em></p>
```

File: 2023/index.md [hunk 3]
```diff
@@ -99,10 +99,113 @@ The 3rd NLP-OSS workshop will be co-located with the EMNLP 2023 conference.
...
+
+<p>Two Decades of the ACL Anthology: Development, Impact, and Open Challenges<br>
+<em>Marcel Bollmann, Nathan Schneider, Arne Ko ̈hn and Matt Post</em></p>
+
+<p>Zelda Rose: a tool for hassle-free training of transformer models<br>
+<em>Loïc Grobol</em></p>
+
+<p>12:15 - 13:45 &nbsp;&nbsp; <strong>Lunch Break </strong> </p>
+<p>13:45 - 14:45 &nbsp;&nbsp; <strong>Invited Talk - StrlX: A Framework for Large Scale Open Source RLHF </strong><br>
+<em>Louis Castricato</em></p>
```
Comment (alvations) [inline 2023/index.md:159]: We've SEA-LION scheduled here.

```
<p>13:45 - 14:55 &nbsp;&nbsp; <strong> Invited Talk - SEA-LION (Southeast Asian Languages In One Network): A Family of Southeast Asian Language Models </strong> <br>
<em>David Ong and Peerat Limkonchotiwat</em></p>
```

**Review by alvations — CHANGES_REQUESTED** [review]
Other than the invited talk changes requested, I think we're good for the rest of the schedule.

Thank you @jgwinnup for putting this up!

**Comment by jgwinnup** [comment]
Ok invited talks have been swapped

**Comment by alvations** [comment]
Thanks Jeremy, LGTM now!

**Pull request merged**